src/core.js

```diff
@@ -822,7 +823,7 @@ $.extend( $.validator, {
 					if ( group ) {
 						$.each( this.groups, function( name, testgroup ) {
 							if ( testgroup === group ) {
-								$( "[name='" + name.replace( /'/g, "\\'" ) + "']", this.currentForm )
+								$( "[name='" + this.escapeCssMeta( name ) + "']", this.currentForm )
```

Comment:
Hi @staabm 
The problem was as I said in my previous comment, `this` inside `$.each( ... )` refers to the current value of the `object|array` we iterate over, not the plugin instance. So this patch should fix it:

```diff
diff --git a/src/core.js b/src/core.js
index 7787a8e..a3d37e9 100644
--- a/src/core.js
+++ b/src/core.js
@@ -843,7 +843,7 @@ $.extend( $.validator, {
 		},
 
 		showLabel: function( element, message ) {
-			var place, group, errorID,
+			var place, group, errorID, v,
 				error = this.errorsFor( element ),
 				elementID = this.idOrName( element ),
 				describedBy = $( element ).attr( "aria-describedby" );
@@ -904,9 +904,10 @@ $.extend( $.validator, {
 					// If this element is grouped, then assign to all elements in the same group
 					group = this.groups[ element.name ];
 					if ( group ) {
-						$.each( this.groups, function( name, testgroup ) {
+						v = this;
+						$.each( v.groups, function( name, testgroup ) {
 							if ( testgroup === group ) {
-								$( "[name='" + this.escapeCssMeta( name ) + "']", this.currentForm )
+								$( "[name='" + v.escapeCssMeta( name ) + "']", v.currentForm )
 									.attr( "aria-describedby", error.attr( "id" ) );
 							}
 						} );
```

Right now, I don't have the time to create a PR with a unit test. But will do soon.

About the test, what can we test to check that this fix work ?